client/src/pages/FiltersPage/FiltersPage.js

```diff
@@ -5,10 +5,27 @@ import Filters from '../../components/Filter/Filter';
 import NavBar from '../../components/Navbar/Navbar';
 import React from 'react';
 import ProfilePage from '../ProfilePage/ProfilePage';
+import Map from 'pigeon-maps';
+import Marker from 'pigeon-marker';
+import Overlay from 'pigeon-overlay';
 
 export default function FiltersPage() {
   const [listOrMap, setListOrMap] = React.useState(true);
 
+  function mapTilerProvider(x, y, z, dpr) {
+    return `https://c.tile.openstreetmap.org/${z}/${x}/${y}.png`;
+  }
+
+  const markers = {
+    place1: [[50.874, 4.6947], 1],
+    place2: [[50.212, 4], 2],
+    place3: [[51, 5], 3],
```

Comment:
rather than using a dictionary of 2D arrays, can we use a 2D array? i.e.
```javascript
const markers = [ [50.874, 4.6947], [50.212, 4], [51,  5] ];
```
this will be more similar to the actual data we'll pull from our database, and then we can just use the index within the array as the `payload`